(GH-1269) Warn when --{sudo}-password is empty (#1393)
puppetlabs/bolt: CHANGELOG.md

# Changelog

## Bolt 1.36.1

> 

Reviewer: ```suggestion
### Bug fixes
```

* **Allow optional `--password` and `--sudo-password` parameters** ([#1269](https://github.com/puppetlabs/bolt/issues/1269))

>   Optional parameters for `--password` and `--sudo-password` were prematurely removed. The previous behaviour of whereby when an argument is not specified for `--password` or `--sudo-password` the user will be prompted for a value. Arguments will be required in a future version.

Reviewer 2: ```suggestion
  Optional parameters for `--password` and `--sudo-password` were prematurely removed. The previous behavior of prompting for a password when an argument is not specified for `--password` or `--sudo-password` has been added back. Arguments will be required in a future version.
```
